src/store/sliceCategories/sliceCategoryType.ts

```diff
@@ -0,0 +1,48 @@
+import {createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+import { LOAD_STATUSES } from '../../constants'
+import {Api,Category } from "api/api";
+
+const api= new Api ();
+export interface State {
+  categories:{categories:Category[]};
+  loadStatus: LOAD_STATUSES;
+}
+
+const initialState: State = {
+  categories:{
+    categories:[
```

Comment:
```suggestion
    categories:[],
```
просто пустой массив как дефолтное значение